features/buyer/requirements.feature

```diff
@@ -136,14 +136,20 @@ Scenario: Create user research participants
    And I don't see the 'Review and publish your requirements' link
 
 @copy
-Scenario: Copy requirements
+Scenario Outline: Copy requirements
```

Comment:
Right? <3
I asked David if there is a way to parametrise tests in Gherkin and it turned out there is one, indeed 😍